python/sdk/merlin/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = "0.25.0"
+VERSION = "0.31.3"
```

Comment:
Somehow this version hasn't been updated for a loooong time. I don't know when we'll cut these SDK changes into a new release, but I've just bumped up this version to the latest tag that we have.

Reply:
Thanks for updating this. This line is actually being updated when GitHub CI releases new SDK (refer to this line: https://github.com/caraml-dev/merlin/blob/main/.github/workflows/release.yml#L40)

Reply:
Ahh... thanks for sharing that! Seems like the last time that GitHub job was run successfully was a year ago 👀 https://github.com/caraml-dev/merlin/actions/workflows/publish-python-sdk.yml 

There's probably something wrong with the GitHub CICD workflow; lemme go fix that

Reply:
Ahhh got it, there's actually no need to update that value 😅 should I just set it as `VERSION = "0.0.0"` to make it obvious it doesn't need to be bumped?

Reply:
-> should I just set it as VERSION = "0.0.0" to make it obvious it doesn't need to be bumped?

That's a good idea. I'm fine with that. 👍🏻